### PR TITLE
Feature: Number input text color

### PR DESCRIPTION
**Pull request by danielevilela**
fixes: #4189 
image

### CODE REVIEW

**Review by arpitnath — CHANGES_REQUESTED** [review]
Please add default value for the widget in `widgetConfig.jsx`

Under definitions, styles.

File: frontend/src/Editor/WidgetManager/widgetConfig.js [hunk 1]
```diff
@@ -1031,6 +1036,7 @@ export const widgets = [
         visibility: { value: '{{true}}' },
         disabledState: { value: '{{false}}' },
         borderRadius: { value: '{{0}}' },
+        textColor: { value: '' },
```
Comment (arpitnath) [inline frontend/src/Editor/WidgetManager/widgetConfig.js:1039]: Can you use the current input color as the default value?
```suggestion
        textColor: { value: '#232e3c' },
```

**Review by arpitnath — CHANGES_REQUESTED** [review]
Can you resolve the merge conflicts too?

**Review by arpitnath — APPROVED** [review]

**Review by arpitnath — APPROVED** [review]

**Comment by gsmithun4** [comment]
deployed https://tj-4193.herokuapp.com/

File: frontend/src/Editor/Components/NumberInput.jsx [hunk 2]
```diff
@@ -6,10 +6,13 @@ export const NumberInput = function NumberInput({
   styles,
   setExposedVariable,
   component,
+  darkMode,
   fireEvent,
 }) {
   const { visibility, borderRadius } = styles;
 
+  const textColor = darkMode && styles.textColor === '#000' ? '#fff' : styles.textColor;
```
Comment (arpitnath) [inline frontend/src/Editor/Components/NumberInput.jsx:14]: As the default color is **#232e3c**, that should be handled, or else on switching from light to dark the input is not legible
```suggestion
  const textColor = darkMode && ['#232e3c', '#000000ff'].includes(styles.textColor) ? '#fff' : styles.textColor;
```

**Review by arpitnath — CHANGES_REQUESTED** [review]

**Review by arpitnath — APPROVED** [review]

**Comment by withshubh** [comment]
Hey @danielevilela 
Thanks for contributing to ToolJet. :raised_hands: 
Please submit the form to claim your Hacktoberfest swag: https://airtable.com/shr1qR6xhqsWO7LD7

**Pull request merged**